# Sent to upstream on 2012-10-23
# By Jesus Rivero <neurogeek@gentoo.org>
diff -uNr opal-3.10.8.orig/include/h323/h323.h opal-3.10.8/include/h323/h323.h
--- opal-3.10.8.orig/include/h323/h323.h        2012-10-22 07:50:37.000000000 -0400
+++ opal-3.10.8/include/h323/h323.h     2012-10-23 14:10:37.000000000 -0400
@@ -40,6 +40,7 @@
 #include <h323/h323con.h>
 #include <h323/gkclient.h>
 #include <opal/buildopts.h>
+#include <ptlib.h>
 
 PString  OpalGetVersion();
 unsigned OpalGetMajorVersion();
